insights/core/dr.py

```diff
@@ -424,6 +424,26 @@ def get_at_least_one(comp):
     return get_requires(component) + get_at_least_one(component)
 
 
+def get_spec_name(c):
```

Comment:
Just a suggestion, but it would be better to add a unit test and coverage test for this new function

And, it seems the `False branch of L441` (441->436) will never be covered.

Reply:
Good suggestion, that was something I intended to do and forgot. So thank you for reminding me.